recipes/f90nml/meta.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+  name: f90nml
+  version: "0.21"
```

Comment:
Please use a jinja variable for the version number to make it easier to maintain the recipe.